api/tests/Api/ECampApiTestCase.php

```diff
@@ -74,8 +74,8 @@ protected static function createClientWithCredentials(?array $credentials = null
         $jwtSignature = substr($jwtToken, $lastPeriodPosition + 1);
 
         $cookies = $client->getCookieJar();
-        $cookies->set(new Cookie('jwt_hp', $jwtHeaderAndPayload, null, null, 'example.com', false, false, false, 'strict'));
-        $cookies->set(new Cookie('jwt_s', $jwtSignature, null, null, 'example.com', false, true, false, 'strict'));
+        $cookies->set(new Cookie('example.com_jwt_hp', $jwtHeaderAndPayload, null, null, 'example.com', false, false, false, 'strict'));
```

Comment:
Might be better if we use localhost instead of example.com to simplify upgrade to api-platform 2.7:

https://github.com/ecamp/ecamp3/pull/2649/files#diff-1c80e80e582855ae7b4bbbdfce8894f47dcb3ef2a8e5093dfef03cba5cebf9c7

Reply:
I see. But then I'd have to change it in .env.testing and probably more tests, which would lead to more conflicts with #2649. So I think switching the tests to localhost (if at all) strictly belongs to #2649.